.github/workflows/ci.yml

```diff
@@ -68,3 +77,10 @@ jobs:
         with:
           working-directory: ./
           run: dub test -c unittest-gtk
+
+      - name: Test DWT Gtk (optimised)
```

Comment:
Isn't this step a duplication now?

Reply:
Indeed, sorry. I've removed the second "Test DWT Gtk" and used the `matrix.buildType` in the first one.